Modules/posixmodule.c

```diff
@@ -14962,6 +14962,326 @@ os_waitstatus_to_exitcode_impl(PyObject *module, PyObject *status_obj)
 #endif
 
 
+#if defined(MS_WINDOWS)
+/*[clinic input]
+os._isdir
+
+    path: 'O'
+
+Return True if path is an existing directory.
+
+This follows symbolic links, so both islink() and isdir() can be true for the
+same path.
```

Comment:
I see no reason to change the existing docstrings of these functions. Docstrings are rarely the same as the online documentation.

```suggestion
Return true if the pathname refers to an existing directory.
```